src/main/java/de/retest/web/WebData.java

```diff
@@ -62,22 +62,28 @@ public Rectangle getOutline() {
 	}
 
 	/**
-	 * @throws NumberFormatException
-	 *             if the given value cannot be converted to integer.
+	 * @throws IllegalStateException
+	 *             if the value for the given key cannot be converted to integer.
 	 */
-	private int toInt( final Object value ) throws NumberFormatException {
-		if ( value instanceof Integer ) {
-			return (Integer) value;
-		}
-		if ( value instanceof String ) {
-			return Integer.parseInt( (String) value );
-		}
-		if ( value instanceof Double ) {
-			return Math.toIntExact( Math.round( (Double) value ) );
-		}
-		if ( value instanceof Long ) {
-			return Math.toIntExact( Math.round( (Long) value ) );
+	public int getAsInt( final String key ) {
+		final Object value = wrappedData.get( key );
+		try {
+			if ( value instanceof Integer ) {
+				return (Integer) value;
+			}
+			if ( value instanceof String ) {
+				return Integer.parseInt( (String) value );
+			}
+			if ( value instanceof Double ) {
+				return Math.toIntExact( Math.round( (Double) value ) );
+			}
+			if ( value instanceof Long ) {
+				return Math.toIntExact( Math.round( (Long) value ) );
+			}
+		} catch ( final Exception e ) {
+			// muted, since we throw a general exception below
```

Comment:
Actually, we do know how to convert the value. However there were some errors doing so. I think this should be thrown.